Singleton/version_2/main.cpp

```diff
@@ -0,0 +1,26 @@
+#include "Singleton.hpp"
+
+void thread1()
+{
+	const std::string	name("David");
+
+    Singleton* singleton = Singleton::getInstacne(name);
+	std::cout << singleton->getName() << " -- thread1\n";
+
+}
+
+void thread2()
+{
+	const std::string	name("Eduard");
```

Comment:
It is better to use curly braces whenever it is possible. You can take it as a rule:
```
const std::string name{"Eduard"};
```

Reply:
Accepted 🤐